Makefile

```diff
@@ -268,6 +268,7 @@ github.com/prometheus/client_golang/prometheus.{NewCounter,NewCounterVec,NewCoun
 NewHistorgram,NewHistogramVec,NewSummary,NewSummaryVec}=github.com/prometheus/client_golang/prometheus/promauto.{NewCounter,\
 NewCounterVec,NewCounterVec,NewGauge,NewGaugeVec,NewGaugeFunc,NewHistorgram,NewHistogramVec,NewSummary,NewSummaryVec}" ./...
 	@$(FAILLINT) -paths "fmt.{Print,Println,Sprint}" -ignore-tests ./...
+	@$(FAILLINT) -paths "sync/atomic" ./...
```

Comment:
Let's add `"sync/atomic=github.com/uber-go/atomic"` to 2 lines above (with histogram) (: No need to create another line

Reply:
Addressed in a422f52f.

I have used go.uber.org/atomic as opposed to github.com/uber-go/atomic because the former is the preferred manner to import the package, see https://github.com/uber-go/atomic#legacy-import-path